.changeset/pink-gifts-count.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We can do without a changeset here since we aren't changing the public interface; `/src/utils` should be all private.